main/src/ca/uwaterloo/flix/language/phase/Parser.scala

```diff
@@ -1498,6 +1510,10 @@ class Parser(val source: Source) extends org.parboiled2.Parser {
       SP ~ keyword("Region") ~ SP ~> ParsedAst.Kind.Region
     }
 
+    def Effect: Rule1[ParsedAst.Kind.Effect] = rule {
+      SP ~ keyword("Effect") ~ SP ~> ParsedAst.Kind.Effect
```

Comment:
I think the "keyword" should just be `Eff` mirroring the `eff` keyword. Internally we can call it `Effect` as you already do.